nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/main/java/org/apache/nifi/processors/mongodb/GetMongo.java

```diff
@@ -85,6 +92,22 @@
             .allowableValues(YES_PP, NO_PP)
             .addValidator(Validator.VALID)
             .build();
+    static final PropertyDescriptor USER_NAME = new PropertyDescriptor.Builder()
+            .name("User Name")
+            .displayName("username")
```

Comment:
I think the `name` and `displayName` values are switched.

Reply:
Thank you for pointing it out. I will add it